{{cookiecutter.github_repository}}/docs/graphql/0-overview.md

```diff
@@ -0,0 +1,64 @@
+# GraphQL
+
+## Authentication
+
+For all auth related requests (login, register etc), clients need to use REST endpoints [mentioned here](../api/1-auth.md).
```

Comment:
This needs to be updated to be in sync with the code changes.

Reply:
Updated.